src/audio/volume/peak_volume.h

```diff
@@ -43,7 +44,11 @@ enum ipc4_peak_volume_param {
 
 enum ipc4_curve_type {
 	IPC4_AUDIO_CURVE_TYPE_NONE = 0,
-	IPC4_AUDIO_CURVE_TYPE_WINDOWS_FADE
+	IPC4_AUDIO_CURVE_TYPE_WINDOWS_FADE,
+	IPC4_AUDIO_CURVE_TYPE_LINEAR,
+	IPC4_AUDIO_CURVE_TYPE_LOG,
+	IPC4_AUDIO_CURVE_TYPE_LINEAR_ZC,
+	IPC4_AUDIO_CURVE_TYPE_LOG_ZC,
```

Comment:
I like this sequence, 0, with windows fade, and then linear.
it brings:
1. compatible with windows.
2. easy to remove later curve type.